src/Components/Authentication/Types.ts

```diff
@@ -34,6 +34,12 @@ export interface FormProps {
   onBackButtonClicked?: (e: Event) => void
 }
 
+interface AfterSignUpAction {
+  action: "save" | "follow" | "editorialSignup"
```

Comment:
The `"editorialSignup"` type isn't used now but that's the keyword I'm going to use for https://artsyproduct.atlassian.net/browse/GROW-858.

Reply:
Makes sense 👍